GenChangelogs.hs

```diff
@@ -24,6 +24,7 @@ main = do
 
   lastDateStr <- last . lines <$> readProcess "git" ["show", "-s", "--format=%cI", "-1", last tags] ""
   lastDate <- zonedTimeToUTC <$> iso8601ParseM lastDateStr
+  print lastDate
```

Comment:
What's this? Is this used?

Reply:
Oh, this is just something I added to debug the script as I had already made a tag, and so it wasn't picking up all the PRs that were merged. I will revert it.